web/api/appsecret.go

```diff
@@ -10,6 +10,7 @@ import (
 	"net/http"
 	"strings"
 
+	"github.com/madappgang/identifo/web/shared"
```

Comment:
shared is not a good naming for this kind of context
Please created 'middlewares' for this particular case